### ISSUE TITLE
Feature Request: List entries on reverse order

### ISSUE THREAD

**Issue by heitorsilva** [jrnl-org/jrnl#396]
When we do a "jrnl -n 3", we get the last three entries on ascending order...

```
01-08-2016 15:00 Entry 1

01-09-2016 15:00 Entry 2

01-10-2016 15:00 Entry 3
```

I would like to see them on descending order, like it is on git log or hg log.

```
01-10-2016 15:00 Entry 3

01-09-2016 15:00 Entry 2

01-08-2016 15:00 Entry 1
```

**Comment by maebert** [comment]
For a simple log using the `--short` command, you could do

```sh
jrnl --short -n 3 | tail -r
```

to reverse the order!

**Comment by heitorsilva** [comment]
I'm on Windows... I have no tail.
=(

That's why I'm proposing reverse order

**Comment by mzfr** [comment]
Will be fixed by #518
